### PR TITLE
Fix links not being clickable in error fields

### PR DESCRIPTION
**Pull request by rayalynn**

### CODE REVIEW

File: dist/css/uikit.css [hunk 1]
```diff
@@ -292,7 +292,8 @@
   .uikit-text-field__label:after {
     margin-left: 3px; }
 
-.uikit-text-field__message {
+.uikit-text-field__message,
+.uikit-text-field__message--split {
```
Comment (mctaggaj) [inline dist/css/uikit.css]: Thus ui looks good but this does not follow BEM

File: dist/css/uikit.css [hunk 2]
```diff
@@ -332,7 +337,8 @@
 .uikit-text-field--error .uikit-text-field__message {
   color: #FB476C; }
 
-.uikit-text-field__message--split {
+.uikit-text-field--error .uikit-text-field__message--split {
+  color: #FB476C;
```
Comment (mctaggaj) [inline dist/css/uikit.css]: This should be a variable

Comment (rayalynn) [reply dist/css/uikit.css]: this is the css file, not scss

Comment (mctaggaj) [reply dist/css/uikit.css]: My bad!

**Pull request merged**